[INFO -- 2018-09-25 17:41:27]
Starting flip_value script
version: 1.1
[INFO -- 2018-09-25 17:41:27]
Initialization strings:
[INFO -- 2018-09-25 17:41:27]
(1) file /tmp/lreg/lreg
[INFO -- 2018-09-25 17:41:27]
(2) set args /tmp/lreg/randInput /tmp/lreg/output
[INFO -- 2018-09-25 17:41:27]
initSignal:0
[INFO -- 2018-09-25 17:41:27]
endSignal:2
[INFO -- 2018-09-25 17:41:27]
faultModel:2
[EXCEPTION -- 2018-09-25 17:41:28]
pythonException: empty range for randrange() (0,0, 0)
[EXCEPTION -- 2018-09-25 17:41:28]
pythonException: empty range for randrange() (0,0, 0)
[INFO -- 2018-09-25 17:41:28]
Backtrace BEGIN:
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:51
#1  0x00007ffff6ec1801 in __GI_abort () at abort.c:79
#2  0x00007ffff7ad38fb in ?? () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#3  0x00007ffff7ad9d3a in ?? () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#4  0x00007ffff7ad9d95 in std::terminate() () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#5  0x00007ffff7ad9fe8 in __cxa_throw () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#6  0x00007ffff7b02f26 in std::__throw_bad_alloc() () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#7  0x00005555555579bc in __gnu_cxx::new_allocator<double>::allocate (this=0x7fffffffd890, __n=18446744073709551601) at /usr/include/c++/7/ext/new_allocator.h:102
#8  0x0000555555557755 in std::allocator_traits<std::allocator<double> >::allocate (__a=..., __n=18446744073709551601) at /usr/include/c++/7/bits/alloc_traits.h:436
#9  0x000055555555722c in std::_Vector_base<double, std::allocator<double> >::_M_allocate (this=0x7fffffffd890, __n=18446744073709551601) at /usr/include/c++/7/bits/stl_vector.h:172
#10 0x0000555555556f4d in std::_Vector_base<double, std::allocator<double> >::_M_create_storage (this=0x7fffffffd890, __n=18446744073709551601) at /usr/include/c++/7/bits/stl_vector.h:187
#11 0x000055555555663d in std::_Vector_base<double, std::allocator<double> >::_Vector_base (this=0x7fffffffd890, __n=18446744073709551601, __a=...) at /usr/include/c++/7/bits/stl_vector.h:138
#12 0x000055555555610f in std::vector<double, std::allocator<double> >::vector (this=0x7fffffffd890, __x=std::vector of length -14, capacity -14 = {...}) at /usr/include/c++/7/bits/stl_vector.h:328
#13 0x000055555555572b in train (iterations=60000, alpha=5.0000000000000002e-05, N=3, stopsWhenStable=true) at lreg.cpp:126
#14 0x0000555555555c26 in main (argc=3, argv=0x7fffffffde68) at lreg.cpp:182

46	../sysdeps/unix/sysv/linux/raise.c: No such file or directory.

Backtrace END
Fliping value of field: std::_Vector_base<double, std::allocator<double> >
Fliping value of field: _M_impl
Fliping value of field: _M_end_of_storage
Fliping a bit of the value pointed by a pointer
Memory content before bitflip:0x000x000x000x000x000x000x000x00
Fault Model: Random bit-flip
base address to flip value: 0x555555771750
address max offset: 8
Memory content after  bitflip:0x230x190x0e0x000x1e0xa20x590xbc
frame name: std::allocator_traits<std::allocator<double> >::allocate
symbol name: y
symbol filename: lreg.cpp
symbol line: 33
value: std::vector of length 20, capacity 32 = {0, 0, 0, 0, 0, 0, 0, 0, -12, -320, -165, -572, 3, -480, -480, -12, -165, -320, -320, -140}
value address: 0x55555575a160 <y>
Type: The type is a structure.
Type sizeof: 24
Field name: std::_Vector_base<double, std::allocator<double> >
Field Type: The type is a structure.
Field Type sizeof: 24

[INFO -- 2018-09-25 17:41:28]
Fault Injection Successful after 1.8542439937591553s
[INFO -- 2018-09-25 17:41:29]
event type: exit
[EXCEPTION -- 2018-09-25 17:41:29]
exit code: no exit code available
